bundles/net.tourbook/src/net/tourbook/weather/WeatherUtils.java

```diff
@@ -0,0 +1,326 @@
+/*******************************************************************************
+ * Copyright (C) 2022 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.weather;
+
+import com.javadocmd.simplelatlng.LatLng;
+import com.javadocmd.simplelatlng.LatLngTool;
+import com.javadocmd.simplelatlng.util.LengthUnit;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.tourbook.Messages;
+import net.tourbook.common.UI;
+import net.tourbook.common.time.TimeTools;
+import net.tourbook.common.time.TourDateTime;
+import net.tourbook.common.util.StringUtils;
+import net.tourbook.common.weather.IWeather;
+import net.tourbook.data.TourData;
+
+public class WeatherUtils {
+
+   /**
+    * @param temperatureValue
+    *           in Celsius
+    * @param windChill
+    *           in Celsius
+    * @param windSpeed
+    *           in km/h
+    * @param windDirection
+    *           in degrees
+    * @param humidity
+    *           in *
+    * @param precipitation
+    *           in mm
+    * @param time
+    *           in epoch seconds
+    * @param timeZonedId
+    *           Example: 17h(15.0°C, feels like 15.0°C, 5.0km/h from 68°, humidity 37%,
+    *           precipitation 0.0mm)
```

Comment:
Wrong location for this comment

![image](https://user-images.githubusercontent.com/1283445/157383418-131714db-3f11-4dad-b24d-d1d14892273f.png)

Reply:
Fixed